Polyfills/Console/Source/Console.cpp

```diff
@@ -36,18 +39,59 @@ namespace
 
     void InvokeCallback(Babylon::Polyfills::Console::CallbackT callback, const Napi::CallbackInfo& info, Babylon::Polyfills::Console::LogLevel logLevel)
     {
-        std::stringstream ss{};
-        for (size_t i = 0; i < info.Length(); i++)
-        {
-            if (i > 0)
-            {
-                ss << " ";
+        
+        std::string formattedString = "";
+        if (info.Length() > 0) {
+            formattedString = info[0].ToString().Utf8Value();
+            // check if this string has substitutions or not
+            std::regex toSub("(%[oOs])|(%(\\d*\\.\\d*)?[dif])");
```

Comment:
Regex are slow, like really slow. There should be a faster test before to check for `%` character before.